src/bin/cratesfyi.rs

```diff
@@ -239,6 +240,9 @@ pub fn main() {
         } else if let Some(_) = matches.subcommand_matches("update-search-index") {
             let conn = db::connect_db().unwrap();
             db::update_search_index(&conn).expect("Failed to update search index");
+        } else if let Some(_) = matches.subcommand_matches("move-to-s3") {
+            let conn = db::connect_db().unwrap();
+            db::file::move_to_s3(&conn, 5_000).expect("Failed to update search index");
```

Comment:
This 5000 value is arbitrarily chosen -- but increasing it much more seems like a bad idea, as it means we're touching that many more rows and attempting to do that much more at the same time. This takes ~5 seconds to run once on my machine (with minio, as I've said, not S3), so running it in a loop won't have too high overhead from process spawning.

Reply:
Is the plan to run the loop in bash, instead? How would we know that it finished?

Reply:
Very loosely, `select count(*) from files where files.content != E'in-s3'` should return no rows. I plan to monitor it somewhat closely while I'm around for the first few hours and, if you'd like, can kill it when I'm not around. I can add the looping logic directly in here, though. Up to you.